charts/posthog/templates/crds/clickhouseoperatorconfigurations.clickhouse.altinity.com.yaml

```diff
@@ -2,6 +2,7 @@
 #
 # NONE
 #
+{{- if .Values.clickhouse.installCRDs }}
```

Comment:
Looking at this change a little closer, I think this is a file that will be overwritten when we we run [`scripts/clickhouse_operator_sync.sh`](https://github.com/PostHog/charts-clickhouse/blob/main/scripts/clickhouse_operator_sync.sh). As such, it would be preferable to update that script over updating them here.

Have a look at the script [here](https://github.com/PostHog/charts-clickhouse/blob/main/scripts/clickhouse_operator_sync.sh#L61). Can you add something similar for the CRDs, perhaps just the addition of the crd files to `FILES`?

@guidoiaquinti what do you think